docs/modules/sql/pages/querying-streams.adoc

```diff
@@ -97,91 +116,190 @@ If the client stops receiving results, but doesn't close the result, Hazelcast w
 
-- `HOP`: A hopping window (sometimes also called _sliding window_), assigns events into overlapping windows of a fixed length. The 4th parameter of this function defines the hop size, i.e. the distance between the starts of two consecutive windows.
-+
-image:ROOT:eventtime-sliding.svg[A hopping window]
+image:ROOT:tumbling-window.svg[A tumbling window]
 
-TIP: For a conceptual introduction to windowing and watermarks, see xref:pipelines:event-time.adoc[].
+The following SQL statement assigns the one-day window and outputs a count of the number of customer orders in each window:
 
 ```sql
-SELECT window_start, window_end, COUNT(*) AS total_trades <1>
+SELECT window_start, window_end, COUNT(*) AS total_orders <1>
 FROM TABLE(TUMBLE(
-  TABLE trades_ordered, <2>
-  DESCRIPTOR(trade_time), <3>
-  INTERVAL '1' MINUTE)) <4>
+  TABLE orders_ordered, <2>
+  DESCRIPTOR(order_time), <3>
+  INTERVAL '1' DAY)) <4>
 GROUP BY 1,2; <5>
 ```
 
-<1> Get a count of all trades that happened in the window.
-<2> Use an ordered view of `tables` as input (explained later).
-<3> Use the timestamp in the `trade_time` column to determine the window the event belongs to.
-<4> Set the size of the tumbling window to one minute.
-<5> Defines the grouping, the numbers `1` and `2` refer to 1st and 2nd column of the `SELECT` clause, therefore we group by the `window_start` and `window_end` columns.
+<1> Get a count of all orders that were submitted in the window.
+<2> <<late-events, Handle late events>>.
+<3> Use the timestamp in the `order_time` column to determine the window the event belongs to.
+<4> Set the size of the tumbling window to one day.
+<5> Defines the grouping, the numbers `1` and `2` refer to 1st and 2nd column of the `SELECT` clause, therefore you group by the `window_start` and `window_end` columns.
 
-When new results are complete for each one-minute window, they are returned:
+New results for each one-day window are only returned when all events that belong to the window have been processed. To learn more about this, see <<late-events>>.
 
 ```
 +-------------------+-------------------+--------------------+
-|window_start       |window_end         |        total_trades|
+|window_start       |window_end         |        total_orders|
 +-------------------+-------------------+--------------------+
-|2022-01-04T00:00   |2022-01-04T00:01   |                  45|
-...
+|2022-01-04T00:00   |2022-01-04T23:59   |                   5|
 ```
 
-=== Creating Watermarks
+[[hopping-windows]]
+=== Hopping Windows
 
-Hazelcast can't emit the result of a windowed aggregation until it has received all the events belonging to the window. But streams typically aren't strictly ordered by time, events arrive each with different latency. To tell Hazelcast how long to wait, you must define a watermark.
+Hopping windows, sometimes called _sliding windows_, also allow you to group events by time. However, hopping windows can overlap. They're useful for taking a snapshot of data. 
```

Comment:
Both tumbling and hopping windows are 'sliding'

Reply:
Removed reference to slide/sliding as I think it might be confusing things.